test/main.js

```diff
@@ -0,0 +1,62 @@
+var utils = require('../');
+var should = require('should');
+var chai = require('chai');
+var expect = chai.expect;
+var gutil = require('gulp-util');
+var File = require('vinyl');
+var es = require('event-stream');
+var fs = require('fs');
+var path = require('path');
+
+var createFile = function(filePath, type) {
```

Comment:
That does seem like a good idea. I could test for it first (reverse these two tests) and then use it throughout the test file.